Controller/ArticleController.php

```diff
@@ -40,5 +41,13 @@ public function ajoutArticleValidation()
 
     }
 
+    public function suppressionArticle($id)
+    {
+        $this->articleManager->getOneArticle($id);
```

Comment:
it's really necessary to recover the article before deleting it ?

Reply:
if so, maybe the suppressionArticle should take the article as a parameter rather than the id ?